packages/flutter/lib/src/material/slider.dart

```diff
@@ -170,12 +170,29 @@ class Slider extends StatefulWidget {
 }
 
 class _SliderState extends State<Slider> with TickerProviderStateMixin {
+  _SliderState() {
+    _reactionController = new AnimationController(
+      duration: kRadialReactionDuration,
+      vsync: this,
+    );
+  }
+
   void _handleChanged(double value) {
     assert(widget.onChanged != null);
     widget.onChanged(value * (widget.max - widget.min) + widget.min);
   }
 
   @override
+  void dispose() {
+    _reactionController?.dispose();
+    super.dispose();
+  }
+
+  // Have to keep the reaction controller here so that we may dispose of it
+  // properly.
```

Comment:
this comment isn't really necessary. I mean, I can't really see any situation in which reading this comment would improve someone's understanding of the code...